internal/pkg/telemetry/fortest.go

```diff
@@ -173,24 +174,24 @@ func (v *forTest) AssertMetrics(t *testing.T, expectedMetrics []metricdata.Metri
 }
 
 type testIDGenerator struct {
-	traceID uint16
-	spanID  uint16
+	traceID atomic.Int64
+	spanID  atomic.Int64
 }
 
 func (g *testIDGenerator) Reset() {
-	g.traceID = 0
-	g.spanID = 0
+	g.traceID.Store(0)
+	g.spanID.Store(0)
 }
```

Comment:
Ako som vsetkym testom nastavil testovaciu telemetriu by default, tak v niektorych testoch bezia nejake operacie paralelne, preto tu musia byt hodnoty osetrene zamkom. V testoch kde spans/metrics assertujeme nie su paralelne operacie.

Reply:
Zisti sa to v CI/lokalne s `go test -race ....`.

Vystup vyzera potom takto:
![image](https://github.com/keboola/keboola-as-code/assets/19371734/58a97564-8946-4640-91da-c2a5d96a1847)